doc/qbk/01_overview.qbk

```diff
@@ -0,0 +1,49 @@
+[/
+    Copyright (c) 2019 Vinnie Falco (vinnie.falco@gmail.com)
+    Copyright (c) 2020 Krystian Stasiowski (sdkrystian@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/cppalliance/json
+]
+
+[section Overview]
+[block'''<?dbhtml stop-chunking?>''']
+
+[$images/logo.png]
```

Comment:
Not necessary. All the HTML files shown will have the Boost logo on the green banner.

Reply:
The thing is Boost.Utility had a different logo and I didn't know where to put it. 

Maybe we could remove the logo entirely but I thought the people who created it wouldn't like that.

Reply:
We can remove the logo entirely. There's no need to show two of them - the one on the green banner is sufficient.

Reply:
Done: dd95ce7f0bfc4faae9ad53c198d7ee11d74a98dc